docs/docs/zh/guide/task/stored-procedure.md

```diff
@@ -7,6 +7,33 @@
    <img src="/img/procedure_edit.png" width="80%" />
  </p>
 
-- 数据源：存储过程的数据源类型支持MySQL和POSTGRESQL两种，选择对应的数据源
-- 方法：是存储过程的方法名称
-- 自定义参数：存储过程的自定义参数类型支持IN、OUT两种，数据类型支持VARCHAR、INTEGER、LONG、FLOAT、DOUBLE、DATE、TIME、TIMESTAMP、BOOLEAN九种数据类型
+- 节点名称：设置任务节点的名称，一个工作流定义中的节点名称是唯一的。
+- 运行标志：标识这个结点是否能正常调度，如果不需要执行，可以打开禁止执行开关。
+- 描述：描述该节点的功能。
+- 任务优先级：worker 线程数不足时，根据优先级从高到低依次执行，优先级一样时根据先进先出原则执行。
+- Worker 分组：任务分配给 worker 组的机器执行，选择Default，会随机选择一台 worker 机执行。
```

Comment:
Your Chinese docs should also use the highlight as English `Default`, just like your doc in English.

Reply:
`Default`